packages/front-end/components/Features/SDKConnections/SDKConnectionsList.tsx

```diff
@@ -80,7 +84,8 @@ export default function SDKConnectionsList() {
             {connections.map((connection) => {
               const hasProxy =
                 !isCloud() && connection.proxy.enabled && connection.proxy.host;
-              const hasCloudProxyForSSE = isCloud() && connection.sseEnabled;
+              const hasCloudProxyForSSE =
+                isCloud() && growthbook?.isOn("proxy-cloud-sse");
```

Comment:
We're not using this feature flag anymore, right?

Reply:
we're using this temporarily until 100% rollout, just so we can show accurate info in the UI